test/validation.spec.ts

```diff
@@ -0,0 +1,56 @@
+global['fetch'] = require('cross-fetch');
+const snapshot = require('../');
+const defaultSpace = require('./examples/space.json');
+
+/*
+## Usage
+`npm run test:validation` // Tests default (basic)
+`npm run test:validation --validation=aave-space-validation`
+*/
+
+const validationArg =
+  process.env['npm_config_validation'] ||
+  (
+    process.argv.find((arg) => arg.includes('--validation=')) ||
+    '--validation=basic'
+  )
+    .split('--validation=')
+    .pop();
+
+const validation = Object.keys(snapshot.utils.validations).find(
+  (s) => validationArg == s
+);
+if (!validation) throw 'Validation function not found';
+const example = require(`../src/validations/${validation}/examples.json`)[0];
+
+describe(`\nTest validation "${validation}"`, () => {
+  let isValid = false;
+  let getValidationTime = null;
+
+  it('Validation should run without any errors', async () => {
+    try {
+      const performanceStart = performance.now();
+      isValid = await snapshot.utils.validations[validation](
+        example.userAddress,
+        example.space || defaultSpace,
+        {},
+        {}
```

Comment:
Would be great to have the validation params from example here
```suggestion
        example.validation.params || {}
```